clarifai/client/client.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, app_id=None, app_secret=None, base_url='https://api.clarifai.
     self.api_info = None
 
   def set_model(self, model):
-    self._model = self._sanitize_param(model, 'default')
+    self._model = self._sanitize_param(model)
```

Comment:
Yeah you're right. Either way, we still shouldn't convert the error to 'default'.